src/components/organisms/AssetActions/Pool/index.tsx

```diff
@@ -22,8 +22,14 @@ import { useWeb3 } from '../../../../providers/Web3'
 import PoolTransactions from '../../../molecules/PoolTransactions'
 import { fetchData, getQueryContext } from '../../../../utils/subgraph'
 
+import { isValidNumber } from './../../../../utils/numberValidations'
+import Decimal from 'decimal.js'
+import { constants } from 'node:buffer'
```

Comment:
I think this is unused throughout this file

Reply:
yep, there are also a lot of variables across the market's code that are never used as well.

Reply:
so then remove then?

Reply:
Should I remove all of them in this PR? It might be better have a create a separate one for them